frontend/src/containers/Login/index.tsx

```diff
@@ -52,8 +54,11 @@ type State = Readonly<typeof initialState>
 class Login extends React.Component<{}, State> {
```

Comment:
@chloe-jiang `Props` should be defined (to have `onAuthSuccess` and `onAuthFailure`) and included in `React.Component`